docrepr/tests/test_output.py

```diff
@@ -166,7 +167,10 @@ def eat_one(self):
 # ---- Helper functions
 
 def _test_cases_to_params(test_cases):
-    return [tuple(test_case.values()) for test_case in test_cases.values()]
+    return [
+        [test_id, *test_case.values()]
```

Comment:
```suggestion
        (test_id, *test_case.values())
```

Is there a reason this was changed from a tuple (per general semantics and Pytest convention) to a list?